roles/pulp_django_user/tasks/user_groups.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Shame there's no squeezer module, but probably not worth the bother of writing one for this.